hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocationServiceSupport.java

```diff
@@ -265,11 +278,16 @@ public ClientMessage getClientMessage() {
     }
 
     private class ResponseThread extends Thread {
-        private final BlockingQueue<ClientPacket> workQueue = new LinkedBlockingQueue<ClientPacket>();
+        private final BlockingQueue<ClientPacket> responseQueue;
 
         public ResponseThread(ThreadGroup threadGroup, String name, ClassLoader classLoader) {
             super(threadGroup, name);
             setContextClassLoader(classLoader);
+
+            IdleStrategy idleStrategy = client.getProperties().getBoolean(FAST)
+                    ? new BackoffIdleStrategy(IDLE_MAX_SPINS, IDLE_MAX_YIELDS, IDLE_MIN_PARK_NS, IDLE_MAX_PARK_NS)
+                    : new NoOpIdleStrategy();
```

Comment:
A `FAST` response queue uses the backoff strategy, and the "regular" one uses busy spinning. What's the reasoning behind this choice?